Practica1/firstDeliverable.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+type data struct {
+	ID   int
+	Name string
+}
+
+func main() {
+	http.HandleFunc("/readCSVFile", handlerReadFile)
+	http.HandleFunc("/searchID", handlerSearchID)
+	fmt.Println("The server is running")
+
+	/*e := http.ListenAndServe(":8090", nil)
+
+	if e != nil {
+		log.Fatal(e)
+	}*/
+	go http.ListenAndServe(":8090", nil)
+
+	res, err := http.Get("http://127.0.0.1:8090/readCSVFile")
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("res.StatusCode: %d\n", res.StatusCode)
+
+	if res.StatusCode != 200 {
+		b, _ := ioutil.ReadAll(res.Body)
+		log.Fatal(string(b))
+	}
+
+	fmt.Println("")
+	fmt.Println("")
+
+	res2, err := http.Get("http://127.0.0.1:8090/searchID")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("res2.StatusCode: %d\n", res2.StatusCode)
+
+	if res2.StatusCode != 200 {
```

Comment:
Avoid using "magic numbers" 
Let's use a constant. Actually the package HTTP, already contains constants: 
`http.StatusOK`